ginga/rv/plugins/Catalogs.py

```diff
@@ -67,13 +71,15 @@ def __init__(self, fv, fitsimage):
         canvas = self.dc.DrawingCanvas()
         canvas.enable_draw(True)
         canvas.set_drawtype(self.drawtype, color='cyan', linestyle='dash')
-        canvas.set_callback('cursor-down', self.btndown)
-        canvas.set_callback('cursor-up', self.btnup)
         canvas.set_callback('draw-event', self.draw_cb)
+        canvas.add_draw_mode('select', down=self.btndown,
+                             move=self.btndown, up=self.btnup)
```

Comment:
@pllim, we might be able to take advantage of a feature that was added not too long ago.  Canvases have a "pick" mode, in addition to "draw" and "edit".  You might be able to use that one instead of defining a new "select" mode.  Just provide checkboxes for "draw" and "pick" (or you can call it "select", but make it put the canvas in "pick" mode).

There are `pick` callbacks which are defined for any canvas object.  In the place where we are adding the object with the "star"-prefixed tag, we can simply register for the "pick-up" callback on the object.  The callback function function signature would look like `(canvas, event, pt)`, where `pt` is a tuple of the point.  In registering for the callback, we can add the object itself (and it's star info) so that the callback looks like: `(canvas, event, pt, obj, starinfo)`.  Then there is no need to check the tag type or extract the info.